NetInventors/sw.ext.neti_tags
# #28251 - Add Tags to Product and Customer streams

This adding Tags at Product and Customer streams.

**Testing:**
1. Go to the Customer/ Product stream and open the Attribute view
2. And now see the wonderful beautiful view from the tags
3. If you don't see it, you are went wrong or you don't reinstalled the tags plugin
4. Now select a Tag und click on the Save button below.
5. If you can't select a tag, you don't created a tag!
6. To create a tag, go to the "Inhalt -> Tags" entry on the Mainline Menu
7. Click on the the button with the plus sign, to create a tag.
8. Enter a name for the Tag into the field "Tag" and click below on the Save button.
9. Now return to Point 4, or you had added also the Tag on the creating Tag view to the Product/ Customer stream.

After this long painful journey to "the adding of the tags", should now listed a Tag under the attribute view.
